Leaf API.postman_environment.json

```
@@ -186,9 +186,39 @@
 			"key": "trimble_application_name",
 			"value": "fill with your app's",
 			"enabled": true
+		},
+		{
+			"key": "raven_client_id",
+			"value": "fill with your app's",
```

We can add a better description for this property
```suggestion
			"value": "fill with the Client ID obtained from Raven",
```

---

Done.